lib/symlink.go

```diff
@@ -1,27 +1,51 @@
 package lib
 
 import (
+	"io"
 	"log"
 	"os"
+	"runtime"
 )
 
-//CreateSymlink : create symlink
+// CreateSymlink : create symlink or copy file to bin directory if windows
 func CreateSymlink(cwd string, dir string) {
+	// If we are on windows the symlink is not working correctly.
+	// Copy the desired terraform binary to the path environment.
+	if runtime.GOOS == "windows" {
+		r, err := os.Open(cwd)
+		if err != nil {
+			log.Fatalf(`Unable to open source binary: %s.`, cwd)
+			os.Exit(1)
+		}
+		defer r.Close()
 
-	err := os.Symlink(cwd, dir)
-	if err != nil {
-		log.Fatalf(`
+		w, err := os.Create(dir + ".exe")
+		if err != nil {
+			log.Fatalf(`Could not create target binary: %s.`, dir+".exe")
```

Comment:
It's the same as https://github.com/warrensbox/terraform-switcher/pull/327#discussion_r1543165942 — are backticks intentional?

Reply:
Let's use double quotes for consistency?